plugins/smartspaces/Makefile.am

```diff
@@ -0,0 +1,18 @@
+# Smart Spaces
+plugindir = $(PLUMA_PLUGINS_LIBS_DIR)
+plugin_PYTHON = \
+	smartspaces.py
+
+plugin_in_files = smartspaces.plugin.desktop.in
+plugin_DATA = $(plugin_in_files:.desktop.in=)
+
+$(plugin_DATA): $(plugin_in_files)
+	$(AM_V_GEN) $(MSGFMT) --keyword=Name --keyword=Description --desktop --template $< -d $(top_srcdir)/po -o $@
+
+EXTRA_DIST = $(plugin_in_files)
+
+CLEANFILES = $(plugin_DATA)
+
+DISTCLEANFILES = $(plugin_DATA)
```

Comment:
the `distclean` target calls `clean` target, so you don't need to add the CLEANFILES file list in DISTCLEANFILES.

Reply:
You can set `DISTCLEANFILES = $(plugin_in_files)` instead.

Reply:
done